.github/workflows/base-release.yaml

```diff
@@ -84,6 +84,32 @@ jobs:
         run: |
           echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
 
+        # otelcol-contrib is built in a separate stage
+      - name: Build ${{ inputs.distribution }}
+        if: inputs.distribution == 'otelcol-contrib'
+        uses: goreleaser/goreleaser-action@9ed2f89a662bf1735a48bc8557fd212fa902bebf # v6.1.0
+        with:
+          distribution: goreleaser-pro
+          version: ${{ env.GORELEASER_PRO_VERSION }}
+          workdir: distributions/${{ inputs.distribution }}
+          args: release --clean --split --timeout 2h --config .goreleaser-build.yaml --release-header-tmpl=../../.github/release-template.md
```

Comment:
What happens when `release --clean --split --timeout 2h --release-header-tmpl=../../.github/release-template.md` is called again below?

Reply:
the build for otelcol-contrib is split between 2 different goreleaser config files as of #666. so the first call gets a different config than the second one.
the first call will just build the binaries, the second one continues business as is, while importing the prebuilt binaries and it should then continue as usual